config/settings.py

```diff
@@ -89,6 +90,16 @@ def pipe_delim(pipe_string):
 }
 DATABASES[decouple_config('DJANGO_DATABASE_KEY')] = decouple_config('DJANGO_DATABASE_URL', cast=db_url)
 
+LIVE_DEMO = decouple_config('LIVE_DEMO', default=False, cast=bool)
```

Comment:
I'd recommend namespacing settings that are specific to Mathesar i.e. `MATHESAR_LIVE_DEMO`, since we may install other Django apps with generic setting names.

Reply:
Good point, will do.